projects/graphql-client/src/utils/builders/request/graphql-request-builder.ts

```diff
@@ -31,7 +31,10 @@ export class GraphQlRequestBuilder {
     return new Map(
       this.originalSelections.map(selection => {
         const key = this.getKeyForSelection(selection);
-        const requestString = `{ ${this.translateRequestFragmentToGql(key, this.mergedRequest[key])} }`;
+        const requestString = `${selection.name ?? selection.path} { ${this.translateRequestFragmentToGql(
```

Comment:
I would need to go review the exact impl - it's been years - but i would guess this will lead to some bugs when account for request merging. I'm not even sure how it would be expected to behave - which request's operation name is used? And if it prevents merging, is that really desirable - to fire multiple requests just to include debug metadata like operation name (maybe better in dev mode but not prod mode)